ui/apps/platform/src/Components/CompoundSearchFilter/components/CompoundSearchFilter.cy.jsx

```diff
@@ -186,4 +186,35 @@ describe(Cypress.spec.relative, () => {
         cy.get(nodeComponenSourceSelectItems).eq(5).should('have.text', 'Dotnet Core Runtime');
         cy.get(nodeComponenSourceSelectItems).eq(6).should('have.text', 'Infrastructure');
     });
+
+    it('should display the date-picker input for the image create time search filter', () => {
+        const config = {
+            Image: imageSearchFilterConfig,
+        };
+
+        setup(config);
+
+        cy.get(selectors.attributeSelectToggle).click();
+        cy.get(selectors.attributeSelectItem('Created Time')).click();
+
+        // The date-picker input should be present
+        cy.get('input[aria-label="Filter by date"]').should('exist');
+
+        // Click on the date-picker toggle
+        cy.get('button[aria-label="Filter by date toggle"]').click();
+
+        // Select a month
+        cy.get('div.pf-v5-c-calendar-month__header-month button').click();
```

Comment:
PF v5 Date Picker doesn't have aria labels for the month select dropdown. The year input has one. Not sure why they decided not to add one for this though 🤔 . Maybe something worth asking them in the PF channel. Will do early next week

Reply:
Weird, could just be an accidental omission. 🤔